continuum/scenarios/scenario_utils.py

```diff
@@ -4,39 +4,123 @@
 from torch.utils.data import DataLoader
 
 from continuum.datasets import InMemoryDataset, H5Dataset
-from continuum.scenarios import ContinualScenario
+from continuum.scenarios import ContinualScenario, OnlineFellowship
 from continuum.tasks import TaskType
 
 
+def _get_remapping_classes_ascending_order(new_classes, current_mapping=None):
+    """
+    Output a vector of classes existing class to get new class labels do:
+     new_label = np.where(new_remapping==old_label)[0][0]
+
+    :param new_classes: list of new classes
+    :param current_mapping: vector with previous remapping
+    """
+
```

Comment:
Stop putting empty line like this after function doc

Reply:
C'est pas corrigé automatiquement par l'automatique code formatting, c'est quoi le probleme? ca fait bugger la documentation automatique? Visuellement ca perturbe rien en vrai....

Reply:
It's a convention, you can check good codes, and there are mostly formatted without that empty line.

Auto formatting is not perfect. Moreover, it's not that hard to remove those lines, and it'll keep the code cleaner, and with a consistent coding style.

Reply:
As an example: if you add an empty line between each line of code, I'm sure your formatter won't fix that, but that's still a bit ugly don't you think?

Reply:
That do not disturb me anyhow... anyway I think I fixed it in last commit.